.github/workflows/check.yml

```diff
@@ -0,0 +1,26 @@
+# Reference: https://github.com/golangci/golangci-lint-action
+name: Check
```

Comment:
Maybe we want to name it just `Linting`? 😁

Reply:
Linting is one type of 'check' others could be 'formatting', 'third party vulnerability check' etc. Hence Check is a workflow with potentially multiple jobs